CMakeLists.txt

```diff
@@ -29,6 +29,10 @@ else (CMAKE_VERSION VERSION_LESS 3.0.0)
 	project(libpointmatcher VERSION ${POINTMATCHER_PROJECT_VERSION})
 endif (CMAKE_VERSION VERSION_LESS 3.0.0)
 
+set(LIB_NAME pointmatcher) # same than PROJECT_NAME, w/o the "lib" prefix
```

Comment:
Two questions:
* Why are you defining this alternate library name?
* If it's necessary, why not re-using PROJECT_NAME?

Reply:
It's named for similarity with the related project libnabo: 
https://github.com/ethz-asl/libnabo/blob/master/CMakeLists.txt#L7-L8

The project is named libpointmatcher, the targets were defined like `add_library(pointmatcher)` (!). 
So, to avoid the repeated use of the two names, I preferred to define that variable. 
For sure, using just PROJECT_NAME would be better, but that would probably require many changes.